docker/README.md

```diff
@@ -1,7 +1,7 @@
 # Welcome to the new Docker based Open Library development environment!
 
 These current Dockerfiles are designed to be an alternative to the previous Vagrant based development environment.
-The setup process and scripts are designed to *NOT* conflict with exisiting Vagrant provisioning, so you should be able to
+The setup process and scripts are designed to _NOT_ conflict with exisiting Vagrant provisioning, so you should be able to
```

Comment:
Can we remove this paragraph mentioning vagrant completely please? We have sunset vagrant support :)